hack/test-extended/default/run.sh

```diff
@@ -57,6 +57,8 @@ API_HOST="${API_HOST:-${DEFAULT_SERVER_IP}}"
 setup_env_vars
 mkdir -p $LOG_DIR $ARTIFACT_DIR
 
+ln -svf `readlink -f $(which openshift)` `dirname $(which openshift)`/atomic-enterprise
```

Comment:
On 7.8.2015 20:23, Clayton Coleman wrote:
> In hack/test-extended/default/run.sh <https://github.com/openshift/origin/pull/3953#discussion_r36546480>:
>
>> @@ -57,6 +57,8 @@ API_HOST="${API_HOST:-${DEFAULT_SERVER_IP}}"
>>  setup_env_vars
>>  mkdir -p $LOG_DIR $ARTIFACT_DIR
>>
>> +ln -svf `readlink -f $(which openshift)` `dirname $(which openshift)`/atomic-enterprise
>
> Remove and replace with change to OPENSHIFT_BINARY_SYMLINKS
>
> —
> Reply to this email directly or view it on GitHub <https://github.com/openshift/origin/pull/3953/files#r36546480>.
>
Nice one. Thanks for pointing this out.